#!/usr/bin/env python
PACKAGE = "basler_bridge"
import roslib;roslib.load_manifest(PACKAGE)
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("confidence_threshold", int_t, 0, "confidence_threshold",  4000, 0, 65536)
gen.add("spatial_filter",   bool_t,   0, "spatial_filte",  True)
gen.add("temporal_filter",   bool_t,   0, "temporal_filter",  True)
gen.add("strength", int_t, 0, "strength",  190, 50, 240)
gen.add("outlier_tolerance", int_t, 0, "outlier_tolerance",  4000, 0, 65536)
gen.add("d0", int_t, 0, "d0",  10, -1, 20)
gen.add("d1", int_t, 0, "d1",  50, 0, 200)
gen.add("d2", int_t, 0, "d2",  5, 0, 50)

exit(gen.generate(PACKAGE, "basler_tof_bridge", "basler_tof_bridge"))
